src/app/app.component.css

```diff
@@ -8,3 +8,5 @@
 .app-container {
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/79989850-01121580-84b9-11ea-8528-36a6297dba7a.png)

Reply:
Search results should be aligned to the centre

Reply:
fixed